### ISSUE TITLE
StaggerVars should include repeatRefresh

### ISSUE THREAD

**Issue by matt-a-t** [greensock/GSAP#595]
https://github.com/greensock/GSAP/blob/f836a3f0002d6ea00f456a461c7e2976e0ec3f38/types/gsap-core.d.ts#L146

I believe that the boolean "repeatRefresh" should be included with StaggerVars. It appears to work but Typescript complains. Including it in this interface should quiet the complaint. If you'd like me to open the PR for it I can.

**Comment by jackdoyle** [comment]
Fair point, we'll get that added in the next release. Thanks for pointing it out.
